rle/internal/rleplus/rleplus.go

```diff
@@ -115,22 +117,27 @@ func Decode(buf []byte) (ints []uint64, err error) {
 				// short block
 				runLength = int(take(4))
 			} else {
-				// long block
-				var buf []byte
-				for {
+				// Modified from the go standard library. Copyright the Go Authors and
```

Comment:
We needed to validate that varints were minimally encoded as well.

Reply:
Previously I was using the multiformats' `go-varint` which was doing that.

Reply:
Not in _this_ file.

Reply:
Yeah, but this is just a reference implementation for testing.

Reply:
The reference implementation should still reject invalid bitfields, right?